presto-main/etc/catalog/prometheus.properties

```diff
@@ -1,5 +1,8 @@
 connector.name=prometheus
-prometheus.uri=http://localhost:9090
-prometheus.query-chunk-duration=10m
-prometheus.max-query-duration=1h
+prometheus.uri=https://52.118.211.217:9090
+prometheus.query-chunk-duration=1d
+prometheus.max-query-duration=2d
 prometheus.cache-ttl=30s
+#prometheus.tls.enabled=true
+#prometheus.tls.truststore-path=/opt/presto/etc/cert/catalog/prometheus-truststore.jks
+#prometheus.tls.truststore-password=changeit
```

Comment:
What are these commended properties here? If we don't need it in default then should we remove these?

Reply:
These are not needed and I removed them.